### PR TITLE
Feat: add translate text function

### PR DESCRIPTION
**Pull request by aknsal**
This PR should close [appwrite/appwrite#4142](https://github.com/appwrite/appwrite/issues/4142)

## Description
Implemented `translate text` function using different providers.
Currently the supoorted providers are :
* Google
* AWS
* Azure

#### Translation using Google
google_proof

#### Translation using Azure
azure_proof

#### Translation using AWS
aws_proof

### CODE REVIEW

File: node/translate_text/src/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+module.exports = async function(req, res) {
+    const {
+      provider,
+      text
+    } = JSON.parse(req.payload);
+
+    let translatedText = "";
+
+    if(provider==='google'){
+      translatedText = await googleProvider(req.payload).catch((e)=>console.log(e));
```
Comment (stnguyen90) [inline node/translate_text/src/index.js:14]: On error, please makes sure to return an error response as detailed in the issue.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Great PR! 🤯 We left some comments during the review, please check them out.

**Comment by Meldiron** [comment]
Thank you so much for the PR 🤩. We're adding the `hacktoberfest-accepted` label to ensure this PR counts towards your Hacktoberfest contributions count. With that said, please stay active on this PR to address any comments once you receive a review. Happy Hacktoberfest! 🎃

**Review by stnguyen90 — APPROVED** [review]
Hey 👋 awesome work on your PR! We've approved your work and it'll be merged soon!

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**